### ISSUE TITLE
enosARM-server-rpi4-latest.img.xz appears to be corrupt

### ISSUE THREAD

**Issue by mbernardi1961** [endeavouros-arm/images#10]
I've tried to download the ARM server image for RPi with both chrome and Opera and neither download could be opened with Xarchiver or copied to SD card with balenaEtcher.
Any chance of a new upload of this file and perhaps a version for RPi5.

**Comment by endeavouros-arm** [comment]
I have created a new image for the RPi 4 / 5 and tested it on both a RPi 4b and RPi 5b and all worked as expected.
For my test platform, I used a x86_64 computer with EndeavourOS and KDE Plasma installed.
I downloaded the image and sha512sum files using Firefox.  Checked the sha512sum. Then burned the image to a
micro SD using gnome-disk-utility.  

Plugged the micro SD into a RPi 5b with keyboard, monitor, ethernet, and a power supply attached.
The RPi booted up, started running the set up program, and upon the second boot the server was working as expected.

Repeated the above with a RPi 4b and got the same results.

Be sure to read the README.md file and follow the instructions carefully.  After install, go to
https://discovery.endeavouros.com/category/arm
and follow the HowTo's to set up a basic home Linux LAN file server.

Are you trying to download the image and burn the image with balenaEtcher with Windows?
That is not impossible, but is problematic.

Let me know how it goes either here or on forum.endeavouros.com 
I am Pudge on the EnOS forum.

Pudge
